### PR TITLE
Update ubuntu-minimal images by GitHub Actions

### PR DESCRIPTION
**Pull request by masa213f**
I want to update ubuntu-minimal images with GitHub Actions.
By this PR, when updating the `18.04/TAG_MINIMAL` or `20.04/TAG_MINIMAL`, `ubuntu-minimal` will be updated.

### CODE REVIEW

**Review by umezawatakeshi — DISMISSED** [review]

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -16,24 +16,54 @@ jobs:
...
+        run: |
+          if [ ! -f COPY_MINIMAL ]; then
+              echo "nothing to pull."
+              exit 0
+          fi
+          TAG_MINIMAL=$(cat TAG_MINIMAL)
+          echo
+          echo "pulling ubuntu-minimal ..."
+          docker pull ubuntu:$TAG_MINIMAL
+          docker tag ubuntu:$TAG_MINIMAL quay.io/cybozu/ubuntu-minimal:$TAG_MINIMAL
```
Comment (zoetrope) [inline .github/workflows/main.yaml:45]: 👀

File: tag_exists [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+TAG="$2"
 
-for t in $(curl -s -H 'Accept-Encoding: application/json' $API/"$NAME" | jq -r '.tags[]|.name'); do
-    if [ "$t" = "$TAG" ]; then
-        echo "ok"
-        exit 0
-    fi
-done
+API=https://quay.io/api/v1
+STATUS=$(curl -s "$API/repository/cybozu/$NAME/tag/$TAG/images" -o /dev/null -w '%{http_code}\n')
```
Comment (zoetrope) [inline tag_exists:12]: 👀

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -16,24 +16,54 @@ jobs:
...
     steps:
       - uses: actions/checkout@v2
-      - name: Extract build targets
+      - name: Extract targets
         run: |
+          TAG_MINIMAL=$(cat TAG_MINIMAL)
+          c="$(../tag_exists ubuntu-minimal $TAG_MINIMAL)"
+          if [ "$c" = ng ]; then
+              echo "copy ubuntu-minimal:$TAG_MINIMAL"
+              echo yes! > COPY_MINIMAL
```
Comment (zoetrope) [inline .github/workflows/main.yaml:25]: What do you mean `COPY_MINIMAL`?
I guess `NEED_UPDATE_MINIMAL`.

**Review by zoetrope — COMMENTED** [review]
LGTM

File: .github/workflows/main.yaml (hunk 3 above)
Comment (masa213f) [reply .github/workflows/main.yaml]: @zoetrope 
Yes. Your guess is correct. I fixed it. 758555c

**Review by zoetrope — APPROVED** [review]
LGTM

**Pull request merged**